volume/drivers/aws/aws.go

```diff
@@ -238,28 +244,50 @@ func (d *Driver) Create(
 		DryRun:           &dryRun,
 		Encrypted:        &encrypted,
 		Size:             &sz,
-		Iops:             iops,
 		VolumeType:       volType,
 		SnapshotId:       snapID,
 	}
+	// Gp2 Volumes don't support the iops parameter
+	if *volType != opsworks.VolumeTypeGp2 {
+		req.Iops = iops
+	}
 	vol, err := d.ec2.CreateVolume(req)
 	if err != nil {
 		dlog.Warnf("Failed in CreateVolumeRequest :%v", err)
 		return "", err
 	}
+
+	volumeID := *vol.VolumeId
+
 	volume := common.NewVolume(
-		*vol.VolumeId,
-		api.FSType_FS_TYPE_NONE,
+		volumeID,
+		api.FSType_FS_TYPE_EXT4,
```

Comment:
Setting EXT4, per @jvinod, the Create path should format and prepare the volume.